### ISSUE TITLE
Have you ever tried cwd without learning the ground truth?

### ISSUE THREAD

**Issue by twmht** [open-mmlab/mmrazor#145]
Hi,

the work (https://github.com/open-mmlab/mmrazor/tree/master/configs/distill/cwd) is impressive, from the implementation, you learn both ground truth and distilling, for example, when talking about cross entropy for classification loss

```
Lce =Lce(cls_score ,y_gt) + Lce(cls_score, teacher_cls_score)
```

but from autoslim(https://arxiv.org/abs/1903.11728), student only learns from teacher, for example

```
Lce = Lce(cls_score, teacher_cls_score)
```

Although from the experiment you use is a single teacher distilling, which is different from autoslim, I am wondering if it's common to learn both ground truth and teacher?

Have you ever tried to learn only teacher soft labels?

**Comment by wutongshenqiu** [comment]
Hi twmht, we have tried learn only from teacher on CenterNet and the result will be a little worse. However, it may be still worth trying that.

Besides, mmrazor also provides such interface, just set `with_student_loss` [here](https://github.com/open-mmlab/mmrazor/blob/master/mmrazor/models/algorithms/general_distill.py#L19) to False in corresponding config.

**Comment by twmht** [comment]
> Hi twmht, we have tried learn only from teacher on CenterNet and the result will be a little worse. However, it may be still worth trying that.

Can you share the config when training only from teacher?  Did you only use cwd loss with classification and fpn?  Did you mimic bounding box regression branch?

**Comment by wutongshenqiu** [comment]
Just add following lines can make student only learn from teacher in the distillation config.

```python
algorithm = dict(
	# ...
	type='GeneralDistill',
	with_student_loss=False,
	# ...
)
```

No, we haven't tried mimic bbox branch yet.

**Comment by twmht** [comment]
thank you.

**Comment by twmht** [comment]
Here are my experience.

I am trying to use cwd in autoslim, I would appreciate if you can give me some advices.

I have tried cwd without classification loss, but with other ground truth loss (like gfl regression), I did not use cwd for fpn, I only use cwd on the classification branch. the result is much worse than use only ground truth. Also, I have to set cwd loss weight to 1 to make it convergence.  but single teacher use much larger loss weight.

I also tried to use cwd with fpn, but the result is still bad.

Also,  I tried to train with ground truth first in warm up stage, and switch to learn only from the teacher, the result is still bad. 

Overall, it's a challenge to make the network converge.

I am looking forward that you can try object detection+distilling with autoslim. it would be very helpful.

**Comment by pppppM** [comment]
We haven't tried `AutoSlim` on downstream tasks yet. 
Currently, we are developing `BigNAS` with all our strengths. 
After `BigNAS`, we will start some experiments(BigNAS & AutoSlim) on downstream tasks. 

For `CWD` in detection with a single teacher, there is some experience to share:

- It can be used on `FPN` and `classification head`, not on `regression head`.

- Adding to `FPN` will be better, but adding it to both `FPN` and `classification head` will not improve further.
